taskana-adapter-camunda-listener/src/test/java/pro/taskana/adapter/camunda/dto/ReferencedTaskTest.java

```diff
@@ -0,0 +1,206 @@
+package pro.taskana.adapter.camunda.dto;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+/** Acceptance test for ReferencedTask setters and getters. */
+public class ReferencedTaskTest {
+
+  ReferencedTask theTask;
+  String theValue;
+
+  public ReferencedTaskTest() {
```

Comment:
I'm not sure if we should forcefully test getter/setter, just to satisfy the code coverage in sonarcloud...maybe we should discuss this in daily/cop...i think last year we decided against it but i could remember this wrong